cmd/fleet/cron.go

```diff
@@ -403,6 +405,25 @@ func cronWorker(
 	}
 	w.Register(jira)
 
+	// create a JiraClient wrapper to introduce forced failures if configured
+	// to do so via the environment variable.
+	var failerClient *worker.TestJiraFailer
+	forcedFailures := os.Getenv("FLEET_JIRA_CLIENT_FORCED_FAILURES")
```

Comment:
Is there any way we could limit the environments that can make use of this? I'd hate for it to be abused / cause accidental issues, however unlikely that may be... Maybe only allow it to do anything based on the config's ServerURL? Worse case we could always remove this part after the load tests.

Reply:
Agree... The alternative is to implement a "JIRA MITM proxy server" that behaves this way, "return error if <criteria> otherwise relay request to real JIRA" (it may be more work though, something to consider near future if we have the need to continue loading this feature).

Reply:
Yeah once we add more integrations it would be nice to have a more general solution like a MITM that we can deploy and configure that way. I think for now I'll add a check based on the config's ServerURL so it contains "fleetdm", which should cover any of our cases.